release-notes.md

```diff
@@ -41,6 +48,74 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v0.6.0
+
+This release has been tracked through the [0.6.0 project](https://github.com/cloud-hypervisor/cloud-hypervisor/projects/7).
+
+Highlights for `cloud-hypervisor` version 0.6.0 include:
+
+### Directly Assigned Devices Hotplug
+
+We continued our efforts around supporting dynamically changing the guest
+resources. After adding support for CPU and memory hotplug, Cloud Hypervisor
+now supports hot plugging and hot unplugging directly assigned (a.k.a. `VFIO`)
+devices into an already running guest. This closes the features gap for
+providing a complete Kata Containers workloads support with Cloud Hypervisor.
+
+### Shared Filesystem Improvements
+
+We enhanced our shared filesystem support through many `virtio-fs` improvements.
+By adding support for DAX, parallel processing of multiple requests, `FS_IO`,
+`LSEEK` and the `MMIO` virtio transport layer to our `vhost_user_fs` daemon, we
+improved our filesystem sharing performance, but also made it more stable and
+compatible with other `virtio-fs` implementations.
+
+### Block and Networking IO Self Offloading
+
+When choosing to offload the paravirtualized block and networking I/O to an
+external process (through the `vhost-user` protocol), Cloud Hypervisor now
+automatically spawns its default `vhost-user-blk` and `vhost-user-net` backends
+into their own, separate processes.
+This provides a seamless parvirtualized I/O user experience for those who want
+to run their guest I/O into separate executions contexts.
+
+### Command Line Interface
+
+More and more Cloud Hypervisor services are exposed through the
+[Rest API](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/vmm/src/api/openapi/cloud-hypervisor.yaml)
+and thus only accessible via relatively cumbersome HTTP calls. In order
+to abstract those calls into a more user friendly tool, we created a Cloud
+Hypervisor Command Line Interface (CLI) called `ch-remote`.
+The `ch-remote` binary is created with each build and available e.g. at
+`cloud-hypervisor/target/debug/ch-remote` when doing a debug build.
+
+Please check `ch-remote --help` for a complete description of all available
```

Comment:
Should edit the GitHub actions file to ship this binary too.

Reply:
Good point.

Reply:
Done.